lib/puppetserver/ca/config/puppet.rb

```diff
@@ -148,6 +161,12 @@ def resolve_settings(overrides = {})
             settings[setting_name] = substitutions[substitution_name] = subbed_value
           end
 
+          cadir = find_cadir(overrides.fetch(:cadir, false),
+                             settings.fetch(:ssldir, '/etc/puppetlabs/puppet/ssl'),
```

Comment:
Does it make sense to use `$confdir/ssl` here instead of hard-coding `/etc/puppetlabs/puppet/ssl`?
I'm assuming that param is what's used if the setting doesn't exist and I'm not really sure how it wouldn't exist since we set it above?

Reply:
It doesn't make sense, but also in thinking through that reply I realized that this default isn't needed either.